_search-plugins/search-pipelines/neural-query-enricher.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: default
+title: Neural query enrich
+nav_order: 12
+has_children: false
+parent: Search processors
+grand_parent: Search pipelines
+---
+
+# Neural query enrich processor
```

Comment:
Should "enrich" be "enricher"?

Reply:
Technically,  yes. I went back and forth on this and decided to go with a more English-like version :) Let's ask @natebower's opinion.

Reply:
Let's err on the side of technical accuracy and go with "enricher".